test/suite/test_compact10.py

```diff
@@ -140,12 +140,12 @@ def test_compact10_full_backup(self):
         assert self.get_files_compacted(uris) == self.num_tables
 
         # Take a second full backup.
-        os.mkdir(self.backup_dir_2)
-        self.take_full_backup(self.backup_dir_2)
+        os.mkdir(self.backup_incr)
```

Comment:
I am not a fan of this change as the name is misleading, we are not doing an incremental backup as the name may suggest.

Reply:
I have changed the naming [3570009](https://github.com/wiredtiger/wiredtiger/pull/9977/commits/3570009210cdd4c711eea7aae6427731f792f903)